client/core/core.go

```diff
@@ -6230,10 +6293,55 @@ func (c *Core) resumeTrades(dc *dexConnection, trackers []*trackedTrade) assetMa
 		if isActive || needsCoins || tracker.unspentContractAmounts() > 0 {
 			relocks.count(wallets.fromAsset.ID)
 		}
+	}
+
+	dc.tradeMtx.Lock()
+	defer dc.tradeMtx.Unlock()
+	for _, tracker := range trackers {
+		// Never overwrite existing trackedTrades.
+		oid := tracker.ID()
+		if _, found := dc.trades[oid]; found {
+			c.log.Tracef("Trade %v already running.", oid)
+			continue
+		}
+		trade := tracker.Trade()
+
+		// Make sure we have the necessary wallets.
+		walletSet, haveAssetConfigs, err := c.walletSet(dc, tracker.Base(), tracker.Quote(), trade.Sell)
+		if err != nil {
+			notifyErr(tracker, TopicWalletMissing, "Wallet retrieval error for active order %s: %v", tracker.token(), err)
+			continue
+		}
+		tracker.wallets = walletSet
+		lockStuff(tracker)
+		dc.trades[oid] = tracker
+
+		mktConf := dc.marketConfig(tracker.mktID)
+
+		if tracker.metaData.EpochDur == 0 { // upgraded with live orders... smart :/
+			if mktConf != nil { // may remain zero if market also vanished
+				tracker.metaData.EpochDur = mktConf.EpochLen
+			}
+		}
+		if tracker.metaData.FromSwapConf == 0 {
+			tracker.metaData.FromSwapConf = walletSet.fromAsset.SwapConf // possibly still zero if !haveAssetConfigs
+		}
```

Comment:
Is seems can just be zero naturally if server sets to zero.

Reply:
I had started looking asset-specific settings to be used by Core but not the wallets, such as a min swap conf setting, but didn't find an nice solution.  We should do this soon though.
Related to a min confs setting per asset, I was looking into a swap conf override (i.e. used to swap or redeem *earlier* than server requires, at your own discretion).  I think we should do that too.  The user should be able to decide to e.g. redeem taker's BTC swap tx after 1 conf instead of maybe a higher server setting of 3, if they want to.